content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -101,161 +101,176 @@ data:
+    `ServiceEntry` for that. Host name of the `ServiceEntry` should be of the form
+    `<name>.<namespace>.global` where name and namespace correspond to the
+    remote service's name and namespace respectively.
+
+    For DNS resolution for services under the
+    `*.global` domain, you need to assign these services an IP address. We
+    suggest assigning an IP address from the 127.255.0.0/16 subnet. These IPs
+    are non-routable outside of a pod. Application traffic for these IPs will
+    be captured by the sidecar and routed to the appropriate remote service.
 
-{{< text yaml >}}
+    > Each service (in the .global DNS domain) must have a unique IP within the
+    cluster.
+
+    {{< text bash >}}
+
+    $ kubectl -n foo apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: ServiceEntry
+    metadata:
+      name: httpbin-bar
+    spec:
+      hosts:
+      # must be of form name.namespace.global
+      - httpbin.bar.global
+      # Treat remote cluster services as part of the service mesh
+      # as all clusters in the service mesh share the same root of trust.
+      location: MESH_INTERNAL
+      ports:
+      - name: http1
+        number: 8000
+        protocol: http
+      resolution: DNS
+      addresses:
+      # the IP address to which httpbin.bar.global will resolve to
+      # must be unique for each remote service, within a given cluster.
+      # This address need not be routable. Traffic for this IP will be captured
+      # by the sidecar and routed appropriately.
+      - 127.255.0.2
+      endpoints:
+      # This is the routable address of the ingress gateway in cluster2 that
+      # sits in front of sleep.bar service. Traffic from the sidecar will be
+      # routed to this address.
+      - address: ${GATEWAY_IP_CLUSTER1}
+        ports:
+          http1: 15443 # Do not change this port value
+    EOF
+    {{< /text >}}
+
+    The configurations above will result in all traffic in `cluster1` for
+    `httpbin.bar.global` on *any port* to be routed to the endpoint
+    `<IPofCluster2IngressGateway>:15443` over an mTLS connection.
+
+    > Do not create a Gateway configuration for port 15443.
+
+    The gateway port 15443 is a special SNI-aware Envoy
+    preconfigured and installed as part of the Istio installation step
+    described in the prerequisite section.  Traffic entering port 15443 will be
+    load balanced among pods of the appropriate internal service of the target
+    cluster (in this case, `httpbin.bar` in `cluster1`).
+
+1. Verify that `httpbin` is accessible from `sleep`.
+
+{{< text bash >}}
+$ kubectl exec $(kubectl -n foo get pod -l app=sleep -o jsonpath={.items..metadata.name}) \
+   -n foo -c sleep -- curl httpbin.bar.global:8000/ip
+{{< /text >}}
+
+## Force remote cluster traffic through egress gateway
+
+If you wish to route all egress traffic from `cluster2` via a dedicated
+egress gateway, use the following service entry for `httpbin.bar`.
```

Comment:
I would add a line here to emphasize that this egress gateway cannot be used for routing other egress traffic. Essentially emphasizing "dedicated" in "dedicated egress gateway".